pnpm-lock.yaml

```diff
@@ -0,0 +1,2352 @@
+lockfileVersion: '6.0'
```

Comment:
We use normal npm for Zipkin server and probably should stay consistent with it here to reduce confusion.

I personally like pnpm a little more than yarn and a lot more than npm so wouldn't mind using it. If you're able to help migrate Zipkin server to pnpm to keep them similar than I think it's fine to add this.

Reply:
Yes. I could help with migrating to pnpm. It was definetely worth it in Etherpad.

Reply:
(notes about pnpm in zipkin proper) sgtm: looks like it is available here, too, but maybe verify darwin+arm64 and help address if anything is busted, because we do our server builds with this https://github.com/search?q=repo%3Aeirslett%2Ffrontend-maven-plugin%20pnpm&type=code

also don't forget to fix any README drift as people may not be used to using it.